src/platform/silabs/efr32/wifi/ethernetif.cpp

```diff
@@ -31,7 +31,7 @@
 #include "FreeRTOS.h"
 #include "event_groups.h"
 #include "task.h"
-#if (SIWX_917 | EXP_BOARD)
+#if (SLI_917_SOC | EXP_BOARD)
```

Comment:
This SIWX_917 define can be removed/replaced. 
Let it be here, there already a cleanup activity pending, can be encountered there.

Reply:
Now using SLI_SI91X_MCU_INTERFACE, will be left here for the cleanup activity.